groovy/libs.gradle/nbproject/org-netbeans-modules-libs-gradle.sig

```diff
@@ -0,0 +1,167 @@
+#Signature file v4.1
+#Version 6.3
```

Comment:
Looks like newly exported API from a library, right @lkishalmi?

Reply:
Well, originally the Gradle Project module embedded the Gradle Tooling library, but as the external Tooling API changes it's signatures as often as not it has been put to its own module, so I do not have to increase the major version of the NetBeans Gradle Project module every time Gradle make a change.